src/main/js/SaveApi.js

```diff
@@ -6,22 +6,23 @@ import { Fetch, getRestUrl, sseService, loadingIndicator, capabilityAugmenter }
 export class SaveApi {
 
     indexRepo(organization, teamName, repoName) {
-        const createUrl = `${getRestUrl({organization})}/pipelines/`;
-
+        const createUrl = repoName ? `${getRestUrl({organization})}pipelines/` : `${getRestUrl({organization, pipeline: teamName})}`;
```

Comment:
Without this, saving to a MBP with github will fail, no? See: https://github.com/jenkinsci/blueocean-pipeline-editor-plugin/blob/master/src/main/js/EditorPage.jsx#L407